src/main/java/com/example/onlinebookshop/repositories/shoppingcart/ShoppingCartRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.example.onlinebookshop.repositories.shoppingcart;
+
+import com.example.onlinebookshop.entities.ShoppingCart;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+public interface ShoppingCartRepository extends JpaRepository<ShoppingCart, Long> {
+    @Query(value = "SELECT cart FROM ShoppingCart cart "
```

Comment:
Or you can just use `@EntityGraph(attributePaths = {"cartItems", "cartItems.book"})`

Reply:
Instead of @Query at all, right?